app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,103 @@
  *= require_tree .
  *= require_self
  */
+
+body {
```

Comment:
It would be better to move this into its own file and require it first.  For example, if you called it `_base.css`.  Then in `application.css`, you'd have:
```
/*
*= require 'base'
*= require_tree .
*/
```